tests/test_cases/test_cocotb/test_cocotb.py

```diff
@@ -27,125 +27,17 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-import contextlib
-import logging
-import re
-import sys
-import textwrap
-import traceback
-import warnings
-from fractions import Fraction
-from decimal import Decimal
-from math import isclose
-
 """
 A set of tests that demonstrate cocotb functionality
 
 Also used as regression test of cocotb capabilities
 """
```

Comment:
```suggestion
"""
Helper functions used by multiple tests in this directory
"""
```

May as well replace the license header with a short one to